Partition (WBP)/Functions/VI0034C.VTI0034.fun.yml

```diff
@@ -0,0 +1,256 @@
+﻿Function:
+  Enabled For RDMLX:  Yes
+
+  Source: |
+    Function Options(*DIRECT) Rcv_List(#wLKData #wLRet)
+    
+    * ******************************************************************************************************
+    * Test Case Description
+    * Issue #34 Test Convert Special Characters in Column Names
+    * The Test Data is setup with the values that all the Fields will have names in secial charactors
+    * And the table attributes of Convert Special Characters in Column Names is set to YES,
+    * table has been recompiled, to have the Convert Special Characters in Column Names to be working.
```

Comment:
This line is superfluous. All tables are compiled in order for them to work.
A comment you put in somewhere had a summary of the functionality. Included the word "CURD", but in Australia we call it "CRUD". Use some or all of that text.

Reply:
You mean I need replace "table has been recompiled" to "All tables are compiled in order for them to work" ?

Reply:
> You mean I need replace "table has been recompiled" to "All tables are compiled in order for them to work" ?

No. Superfluous means not needed.

A comment you put in somewhere had a summary of the functionality. Included the word "CURD", but in Australia we call it "CRUD". Use some or all of that text.

Reply:
Updated the comment section text as "CRUD".
Removed not needed comment in the code.